sale_force_invoiced/models/sale_order.py

```diff
@@ -24,3 +24,14 @@ def _compute_invoice_status(self):
             lambda so: so.force_invoiced and so.state in ("sale", "done")
         ).update({"invoice_status": "invoiced"})
         return res
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    @api.depends("order_id.force_invoiced")
```

Comment:
Original dependencies are : 

``    @api.depends('state', 'price_reduce', 'product_id', 'untaxed_amount_invoiced', 'qty_delivered', 'product_uom_qty')
``

I guess you should write : 
``    @api.depends('state', 'price_reduce', 'product_id', 'untaxed_amount_invoiced', 'qty_delivered', 'product_uom_qty', 'order_id.force_invoiced')
``
Don't you think ?

Reply:
No, they are accumulative. You don't need to rewrite all of them.

Reply:
oh ! thanks, I didn't know !